src/test/ui/runtime/out-of-stack.rs

```diff
@@ -33,7 +33,7 @@ fn loud_recurse() {
     black_box(()); // don't optimize this into a tail call. please.
 }
 
-#[cfg(unix)]
+#[cfg(all(unix, not(target_os = "fuchsia")))]
```

Comment:
We should just `// ignore-fuchsia` these two with a `FIXME` comment (like Android). We may be able to fix these tests if we can get the right signal from zircon and translate it back to SIGABRT.

Reply:
The ignore-android cites #20004 which is from a spurious test failure; I think this can work on Fuchsia.